runners/direct-java/src/test/java/org/apache/beam/runners/direct/portable/artifact/LocalFileSystemArtifactStagerServiceTest.java

```diff
@@ -88,7 +88,11 @@ public void singleDataPutArtifactSucceeds() throws Exception {
     String name = "my-artifact";
     requestObserver.onNext(
         ArtifactApi.PutArtifactRequest.newBuilder()
-            .setMetadata(ArtifactApi.ArtifactMetadata.newBuilder().setName(name).build())
+            .setMetadata(
```

Comment:
In this file we only set the token - should we also verify it somewhere or it's not needed?